components/Schema.js

```diff
@@ -103,12 +103,19 @@ function SchemaProp({ prop, propName, required = false, path = '', description =
 
 function SchemaPropRow({ prop, propName, required = false, path = '', description = '', isCircular = false }) {
   const acceptedValues = prop.enum() && prop.enum().length ? prop.enum().join(', ') : '_Any_';
-  const types = `${prop.anyOf() ? `anyOf` : ''}${prop.allOf() ? `allOf` : ''}${prop.oneOf() ? `oneOf` : ''}${prop.items() && !Array.isArray(prop.items()) && prop.items().type() ? `${prop.items().type()}` : ''}`;
-  description = `${description || prop.description() || ''}${isCircular ? ' **[CIRCULAR]**': ''}`
+  const types = [
+      prop.type(),
+      prop.anyOf() && `anyOf`,
+      prop.allOf() && `allOf`,
+      prop.oneOf() && `oneOf`,
+      prop.items() && !Array.isArray(prop.items()) && prop.items().type()
```

Comment:
I forgot that `type()` can be array of types. I think that this code is valid, but please check it :) 

```suggestion
      Array.isArray(prop.type()) ? `(${prop.type().join(' or ')})` : prop.type(),
      prop.anyOf() && `anyOf`,
      prop.allOf() && `allOf`,
      prop.oneOf() && `oneOf`,
      prop.items() && !Array.isArray(prop.items()) && prop.items().type() && Array.isArray(prop.items().type()) ? `(${prop.items().type().join(' or ')})` : prop.items().type()
```

Reply:
Sorry, rejected 😅